packages/create-sitecore-jss/src/templates/node-headless-ssr-experience-edge/config.js

```diff
@@ -0,0 +1,42 @@
+const appName = process.env.SITECORE_JSS_APP_NAME;
```

Comment:
Do we want to support ejs here?
```suggestion
const appName = process.env.SITECORE_JSS_APP_NAME || <%- appName %>;
```

Reply:
Resolving these as per @ambrauer's answer in the other PR